src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const th = document.querySelectorAll('th');
+const td = document.querySelectorAll('td');
+const columnsAmount = document.querySelector('thead').rows[0].cells.length;
+
+function cloneCells(arg, step) {
```

Comment:
```suggestion
function cloneCells(elements, step) {
```

pay attention to naming next time, use more descriptive names 🙂